README.md

```diff
@@ -77,7 +77,29 @@ The following steps can be done via SSH or directly on the robot (SSH: `ssh admi
 1. Close the current hero-display session
 2. Repeat steps 1-3 & 5 from [First install](#first-install)
 
+
+## HERO1
+### Services
+The following services are running on HERO1. Normally these are runned from the demo account:
+- **hero1-battery-conversion**: Publishing a BatteryState message from the custom Toyota messages
+- **hero1-monitor.service**: HERO version of `hsrb_monitor` from `hsrb_bringup`
+- **hero1-ntpdate.service**: Service to sync time with servers
+- **hero1-phone-ui.service**: Service to always have the webserver running
+- **hero1-roscore.service**: Service containing the roscore
```

Comment:
technically this launches more than just the roscore but I would need to check the toyota code to see what it does exactly

Reply:
Added more info